Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,33 @@
+$image-logo: './assets/wolox_logo_white.svg';
+
+.footer {
+  background-color: $blue;
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 10px;
```

Comment:
why do you need this? this leaves an empty space at the bottom and you don't want that.